configmap-reload.go

```diff
@@ -86,20 +184,61 @@ func main() {
 		os.Exit(1)
 	}
 
+	// why template without secrets?
```

Comment:
if we plan to pass env vars (seems like the more common use case here) we can't assume `templateFile` and `secretPath` are so tightly coupled